terraform/aws/components/nhais-responder/data_secrets.tf

```diff
@@ -0,0 +1,39 @@
+data "aws_secretsmanager_secret" "mq_username" {
+  name = "amazon-mq-nia-broker-username"
+}
+
+data "aws_secretsmanager_secret" "mq_password" {
+  name = "amazon-mq-nia-broker-password"
+}
+
+data "aws_secretsmanager_secret" "docdb_master_username" {
+  name = "docdb-master-username"
+}
+
+data "aws_secretsmanager_secret" "docdb_master_password" {
+  name = "docdb-master-password"
+}
+
+data "aws_secretsmanager_secret" "nhais_mesh_mailbox_id" {
```

Comment:
All `nhais_mesh...` secrets needs to be `nhais_responder_mesh...` instead. Some values COULD be shared but I think it would be more consistent if they are separate.

You only need to create secrets for "vp" - the responder will not be deployed into any other environment.

For now use the came values as for the `nhais_vp...` secrets. I can help adjust them and document that when we go to test this.

Reply:
fixed